src/components/TitlesView/TitlesView.js

```diff
@@ -22,14 +31,71 @@ const TitlesViews = ({
     ISSN,
     ISBN,
   } = indentifierTypeNames;
+  const createInstanceLink = (row) => {
+    const identifiersValues = getIdentifiersValues(row.identifiers, [ISBN, ISSN], identifierTypesById);
+    const isbn = identifiersValues?.[ISBN];
+    const issn = identifiersValues?.[ISSN];
+
+    const advancedSearchRows = [
+      {
+        bool: '',
+        match: 'exactPhrase',
```

Comment:
There's a `ADVANCED_SEARCH_MATCH_OPTIONS` exported constant in `@folio/stripes/components` that you can use here.

Reply:
done